cmd/lp-cc/main.go

```diff
@@ -0,0 +1,144 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+	"strings"
+
+	_ "net/http/pprof"
+
+	"github.com/ScottSallinen/lollipop/enforce"
+	"github.com/ScottSallinen/lollipop/framework"
+	"github.com/ScottSallinen/lollipop/graph"
+)
+
+func info(args ...any) {
+	log.Println("[CC]\t", fmt.Sprint(args...))
+}
+
+func EdgeParser(lineText string) graph.RawEdge[EdgeProperty] {
+	stringFields := strings.Fields(lineText)
+
+	sflen := len(stringFields)
+	enforce.ENFORCE(sflen == 2 || sflen == 3)
+
+	src, _ := strconv.Atoi(stringFields[0])
+	dst, _ := strconv.Atoi(stringFields[1])
+
+	weight := 1.0
+	var err error
+	if sflen >= 3 {
+		weight, err = strconv.ParseFloat(stringFields[2], 32)
+		enforce.ENFORCE(err, "Text file parse error: weight not floats?")
+	}
```

Comment:
```suggestion
```

Remove the weight, it is not necessary for CC.

Reply:
Fixed.